appcues/src/main/java/com/appcues/AppcuesExperienceActions.kt

```diff
@@ -0,0 +1,68 @@
+package com.appcues
+
+import com.appcues.analytics.AnalyticsTracker
+import com.appcues.data.model.RenderContext
+import com.appcues.data.model.StepReference.StepOffset
+import com.appcues.ui.ExperienceRenderer
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+/**
+ * TODO document this
+ */
+public class AppcuesExperienceActions internal constructor(
```

Comment:
figure out how to make this an instance per custom component (so that we can have the list of actions from that component)